.github/CODEOWNERS

```diff
@@ -65,7 +65,7 @@ core/scripts/gateway @smartcontractkit/dev-services
 /core/scripts/keystone @smartcontractkit/keystone
 
 # Contracts catch all, for files not matched by the more specific patterns below
-/contracts/ @RensR @matYang @RayXpub @elatoskinas
+/contracts/ @smartcontractkit/core-solidity
```

Comment:
![image](https://github.com/user-attachments/assets/a674bf1e-3ef8-4448-bad4-109d955fae4e)

Reply:
https://github.com/orgs/smartcontractkit/teams/core-solidity

Reply:
Yes we are good to go 🙂